templates/migration/templates/migrationservice-configmap.yaml

```diff
@@ -18,6 +18,14 @@ data:
   USERS_PUBLIC_PLAN_TECHNICAL_PASSWORD: "{{ .Values.publicPlanTechnicalPassword }}"
   USERS_PUBLIC_PLAN_TECHNICAL_EMAIL: "{{ .Values.publicPlanTechnicalEmail }}"
   USERS_PUBLIC_PLAN_TECHNICAL_TENANTID: "{{ .Values.publicPlanTechnicalTenantId }}"
+
+  WEBLATE_CONFIG_SERVER_URL: "{{ .Values.weblateServerUrl }}"
```

Comment:
Could you please use :

weblate.server.url
weblate.project.name
weblate.project.slug
weblate.project.webUrl
weblate.usecustomResourcesPath
weblate.customResourcePath

or prefix it with the migration.weblate.* if the configs is only for migrations service.
The config in the values.yaml will look much cleaner:

```
migration:
    weblate:
        server:
            url: "..."
        project:
            name: "..."
            slug: "..."
...
```

and you have to add a default config to the values.yaml of this project because of the subtree:
```
migration:
    weblate:
        server: {}
        project: {}
```